ext/civicrm_admin_ui/managed/SavedSearch_Administer_Location_Types.mgd.php

```diff
@@ -0,0 +1,170 @@
+            [
+              'type' => 'field',
+              'key' => 'name',
+              'dataType' => 'String',
+              'label' => E::ts('Name'),
+              'sortable' => TRUE,
+              'title' => 'Location Type ID: [id]',
+            ],
+            [
+              'type' => 'field',
+              'key' => 'display_name',
+              'dataType' => 'String',
+              'label' => E::ts('Display Name'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'vcard_name',
+              'dataType' => 'String',
+              'label' => E::ts('vCard'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'description',
+              'dataType' => 'String',
+              'label' => E::ts('Description'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_active',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Enabled?'),
+              'sortable' => TRUE,
+              'editable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_default',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Default?'),
+              'sortable' => TRUE,
+            ],
+            [
+              'size' => 'btn-sm',
+              'links' => [
+                [
+                  'path' => '',
+                  'icon' => 'fa-pencil',
+                  'text' => E::ts('Edit'),
+                  'style' => 'default',
+                  'condition' => [],
+                  'entity' => 'LocationType',
+                  'action' => 'update',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                ],
+                [
+                  'entity' => 'LocationType',
+                  'action' => 'delete',
+                  'join' => '',
+                  'target' => 'crm-popup',
+                  'icon' => 'fa-trash',
+                  'text' => E::ts('Delete'),
+                  'style' => 'danger',
+                  'path' => '',
+                  'condition' => [
+                    'is_reserved',
+                    '=',
+                    FALSE,
+                  ],
+                ],
+              ],
+              'type' => 'buttons',
+              'alignment' => 'text-right',
+            ],
+          ],
+          'addButton' => [
+            'path' => 'civicrm/admin/locationType/edit?action=add&reset=1',
+            'text' => E::ts('Add Location Type'),
+            'icon' => 'fa-plus',
+          ],
+          'cssRules' => [
+            [
+              'disabled',
+              'is_active',
+              '=',
+              FALSE,
+            ],
```

Comment:
Great! CSS rule is a nice touch. We should make sure the others do this as well.

Reply:
The buttons don't look great when disabled, but I think that's a separate issue.

![image](https://user-images.githubusercontent.com/13518928/195904649-6fc12028-5f2c-4a83-9f2c-94148aa845f1.png)

Reply:
I've created an issue for this: https://lab.civicrm.org/dev/core/-/issues/3916